src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceChannelFactory.cs

```diff
@@ -450,8 +482,7 @@ protected override void OnEndClose(IAsyncResult result)
 
             protected internal override Task OnCloseAsync(TimeSpan timeout)
```

Comment:
I just checked and this class isn't public so we're safe to remove the dead code paths.